src/box/alter.cc

```diff
@@ -4425,6 +4425,15 @@ on_replace_cluster_clear_id(struct trigger *trigger, void * /* event */)
 	return 0;
```

Comment:
Nit: commit msg typo: ~~the~~ during the join process

Reply:
Fixed.